baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +300,72 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality.
+
+### Policies
+
+#### GWS.CHAT.7.1v0.1
+Chat Content Reporting SHALL be Enabled.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+#### GWS.CHAT.7.2v0.1
+All Conversation Types SHALL be Selected.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+  #### GWS.CHAT.7.3v0.1
+Spam, Confidential Information, and Sensitive Information SHALL be Selected.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
```

Comment:
The current rationale doesn't tie very well to the policy statement. Users will be able to report any message, regardless of the categories selected here. What selecting a category does is allow additional granularity in how they can categorize a given message. Perhaps most importantly, it allows them to see that you're interested in receiving reports about spam, confidential information, and sensitive information.

```suggestion
  - Users may be uncertain what kind of messages should be reported. Selecting these message types may help users infer that these types of messages should be reported.
```